test/libsolidity/smtCheckerTests/operators/userDefined/user_defined_operator_matches_equivalent_function_call.sol

```diff
@@ -70,7 +70,7 @@ contract C {
 // Warning 6328: (2173-2199): CHC: Assertion violation might happen here.
 // Warning 6328: (2209-2235): CHC: Assertion violation might happen here.
 // Warning 6328: (2245-2271): CHC: Assertion violation might happen here.
-// Info 1391: CHC: 10 verification condition(s) proved safe! Enable the model checker option "show proved safe" to see all of them.
+// Info 1391: CHC: 13 verification condition(s) proved safe! Enable the model checker option "show proved safe" to see all of them.
```

Comment:
I don't know where that extra check comes from.
On my local PC I keep getting 12 here, but on CI it prints 13.

Reply:
13 is the correct one; 1 BMC check changed into CHC and 2 `might happen` changed into verified checks.